Sources/Algorithms/LazySplitCollection.swift

```diff
@@ -0,0 +1,330 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift Algorithms open source project
+//
+// Copyright (c) 2021 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+//
+//===----------------------------------------------------------------------===//
+
+/// A collection that lazily splits a base collection into subsequences separated by elements that satisfy the
```

Comment:
Nit: Really, really a nit, but I think is a code style guide we try to maintain 80 columns max per line. 
So as the comment in line 10 marks the limit :)

Reply:
Funny/sad story: I actually did go through the file and manually reformat all these comments so they were visually wrapped to those comment lines. But I did it in Xcode, which of course renders triple-slash comments in a smaller font. 🤦‍♂️

I'm very happy to follow code style guidelines. I'd looked around a few times for a formatter so it would be painless, but hadn't had much luck. I was just searching again now and found https://github.com/apple/swift-format. I don't know why it eluded me before. Going to give it a try.

Reply:
Mostly done. `swift-format` doesn't seem to reformat doc comments for line length, at least with default config. I reformatted most comments manually, but there are still a few with lines longer than 80 in code examples that I didn't want to tackle yet.